src/components/App.css

```diff
@@ -216,7 +226,7 @@ input[type="number"] {
   justify-content: space-between;
   flex-direction: column;
   margin-top: 10px;
-  height: 100%;
+  height: 314px;
```

Comment:
The height on `.battle-toolbar--options-dropdown` and `.battle-toolbar--options-container` do not seem to be necessary. The dropdown takes whatever height it needs to contain the buttons.